src/sentry/static/sentry/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Link from 'app/components/links/link';
+import {PanelItem} from 'app/components/panels';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import space from 'app/styles/space';
+import {Organization, SentryApp} from 'app/types';
+import {t} from 'app/locale';
+
+import IntegrationStatus from './integrationStatus';
+
+type Props = {
+  organization: Organization;
+  type: 'plugin' | 'provider' | 'sentry-app';
+  slug: string;
+  displayName: string;
+  status: 'Installed' | 'Not Installed' | 'Pending';
+  publishStatus: 'unpublished' | 'published' | 'internal';
+  installations: number;
+};
+
+const urlMap = {
+  plugin: 'plugins',
+  provider: 'integrations',
+  'sentry-app': 'sentry-apps',
+};
+
+const IntegrationRow = (props: Props) => {
+  const {
+    organization,
+    type,
+    slug,
+    displayName,
+    status,
+    publishStatus,
+    installations,
```

Comment:
@NisanthanNanthakumar I think it would be better to just call this `configurations` and pass in `0` for Sentry Apps.  Passing in `1` when it's installed doesn't actually matter since we never use the value for Sentry Apps.